appservice/src/intTest/groovy/nu/yona/server/UpdateBuddyTest.groovy

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server
+
+import groovy.json.*
+import nu.yona.server.test.User
+
+class UpdateBuddyTest extends AbstractAppServiceIntegrationTest
```

Comment:
I renamed the test name to `UpdateBuddyUserInfoTest`. I don't like to add it to `BasicBuddyTest` because it is already so long (and hard to find it there later).